### PR TITLE
Ps icon refactor

### PR DESCRIPTION
**Pull request by psantos1113**
- Update the Font Awesome icon usage to the FontAwesomeIcon React component to make it consistent. 
- Change the icon paths for slack and jira to have the icon set name before the icon name i.e. 'fab/sack' and 'fab/jira' 
- Create a utility to construct the array that FontAwesomeIcon needs to load branded icons correctly.
- Created the alert-icon CSS class to add space between the FontAwesomeIcons and text elements.

### CODE REVIEW

File: src/main/js/distribution/ProjectConfiguration.js [hunk 1]
```diff
@@ -3,13 +3,19 @@ import PropTypes from 'prop-types';
...
 }
 
 function assignDataFormat(cell, row) {
-    const cellContent = (row.missing) ? <span className="missingBlackDuckData"><span className="fa fa-exclamation-triangle fa-fw" aria-hidden="true" />{cell}</span> : cell;
+    const cellContent = (row.missing) ? <span className="missingBlackDuckData">
+        <span className="fa-layers fa-fw">
+            <FontAwesomeIcon icon="exclamation-triangle" className="alert-icon" size="lg" />{cell}
+        </span> </span>
+        : cell;
+
```
Comment (bamandel) [inline src/main/js/distribution/ProjectConfiguration.js]: Can we format this differently? It's a bit hard to read at the moment. I'd recommend:
`const cellContent = (row.missing) ?
    <span className="missingBlackDuckData">
        <span className="fa-layers fa-fw">
            <FontAwesomeIcon icon="exclamation-triangle" className="alert-icon" size="lg" />{cell}
        </span>
     </span>
        : cell;`

Or something similar. It's a very large ternary operator, so even a variable for the HTML might be a better option.

Comment (bamandel) [reply src/main/js/distribution/ProjectConfiguration.js]: That did not format at all. So I think another variable for the HTML might make this easier to read.

File: src/main/js/util/iconUtility.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    if (iconPathData) {
+        path = ['fas'];
+        if (iconPathData.includes('/')) {
+            path = iconPathData.split('/');
+        } else {
+            path.push(iconPathData);
+        }
+    }
+    return path;
+}
```
Comment (bamandel) [inline src/main/js/util/iconUtility.js]: Another option is to create our own fontawesome component that wraps the original and adds this functionality when displaying the field.

Comment (psantos1113) [reply src/main/js/util/iconUtility.js]: I tried that last week and was concerned about the number of properties our wrapper would need to support.  I have another idea.

**Pull request merged**